### ISSUE TITLE
Frontend bug: missing task on run page

### ISSUE THREAD

**Issue by joaquinvanschoren** [openml/OpenML#774]
Example: https://www.openml.org/r/8867786
This happens when the task is supervised but the underlying dataset does not have a default target set.

**Comment by janvanrijn** [comment]
This problem does not occur if the task is created by the api. I asked this
many times before, let us PLEASE disable the frontend task creation. It's
extremely buggy and creates a lot of inconsistent data in the database.


Op dinsdag 7 augustus 2018 heeft Joaquin Vanschoren <
notifications@github.com> het volgende geschreven:

**Comment by joaquinvanschoren** [comment]
Does the API not allow you to create a supervised task if the underlying dataset does not have a default target feature? In this case, the data in the database is actually consistent? The task is created correctly and has the correct target feature.

**Comment by janvanrijn** [comment]
> Does the API not allow you to create a supervised task if the underlying dataset does not have a default target feature?

Yes, it does. However, the api fn requires users to specify a target attribute. 

> In this case, the data in the database is actually consistent? 

Why? 

> The task is created correctly and has the correct target feature.

Can you point out which tasks?

**Comment by joaquinvanschoren** [comment]
Fine, I'll shut it down. Note however that users will have no other way to create tasks.
There is no support in the Python and R API. Maybe the Java API can do it, but there is no documentation or example on how to do it.

**Comment by joaquinvanschoren** [comment]
Sure, Just check the XML (or JSON) of the run: https://www.openml.org/api/v1/run/8867786
Here are the task details: https://www.openml.org/api/v1/task/167127

**Comment by janvanrijn** [comment]
Seems like the user created the task in the correct way, by specifying the target feature manually. 

> Fine, I'll shut it down. Note however that users will have no other way to create tasks.
There is no support in the Python and R API. Maybe the Java API can do it, but there is no documentation or example on how to do it.

I thought we agreed earlier that the form would be attached to the API, such that the current code does no longer need to be executed, but the API handles the creation of tasks. At/After the Porto workshop I spend a lot of time to make the API robust and strict when it comes to task creation (see: https://github.com/openml/OpenML/blob/master/openml_OS/models/api/v1/Api_task.php#L235), it feels silly to not take advantage of this.
